docs/tutorial.rst

```diff
@@ -28,18 +28,17 @@ And even more ways to say that you want to look for restaurants:
 - `I'm in the North of town and I want chinese food`
 - `I'm hungry`
 
-The first job of rasa NLU is to assign any given sentence to one of the categories: ``greet``, ``restaurant_search``, or ``thankyou``. 
+The first job of rasa NLU is to assign any given sentence to one of the **intent** categories: ``greet``, ``restaurant_search``, or ``thankyou``. 
 
-The second job is to label words like "Mexican" and "center" as ``cuisine`` and ``location`` entities, respectively. 
+The second job is to label words like "Mexican" and "center" as ``cuisine`` and ``location`` **entities**, respectively. 
 In this tutorial we'll build a model which does exactly that.
 
 Preparing the Training Data
 ---------------------------
 
-The best way to get training data is from *real users*, and the best way to do that is to `pretend to be the bot yourself <https://conversations.golastmile.com/put-on-your-robot-costume-and-be-the-minimum-viable-bot-yourself-3e48a5a59308#.d4tmdan68>`_. But to help get you started we have some `data saved <https://github.com/RasaHQ/rasa_nlu/blob/master/data/examples/rasa/demo-rasa.json>`_
-
-Download the file and open it, and you'll see a list of training examples like these:
+The training data are essential to develop chatbots. They should include texts to be interprated and the structured data (intent/entities) we expect chatbots to convert the texts into. The best way to get training texts is from *real users*, and the best way to get the structured data is to `pretend to be the bot yourself <https://conversations.golastmile.com/put-on-your-robot-costume-and-be-the-minimum-viable-bot-yourself-3e48a5a59308#.d4tmdan68>`_. But to help get you started, we have some `data saved <https://github.com/RasaHQ/rasa_nlu/blob/master/data/examples/rasa/demo-rasa.json>`_.
```

Comment:
Sorry about the typo. Regarding "data", when I publish articles in the bioinformatics field, I think we can consider this word either a plural form of datum or an uncountable noun meaning information, depending on contexts. So, the both for me are good, but I agree with and respect your choice.